Sources/Controllers/MessagesViewController.swift

```diff
@@ -59,6 +61,18 @@ UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
         return false
     }
 
+    /// A CGFloat value that adds to (or, if negative, subtracts from) the automatically
+    /// computed value of `messagesCollectionView.contentInset.bottom`. Meant to be used
+    /// as a measure of last resort when the built-in algorithm does not produce the right
+    /// value for your app. Please let us know when you end up having to use this property.
+    open var extraBottomInset: CGFloat = 0 {
```

Comment:
Do you think we can support an `additionalTopInset` as well? This can be used to allow users to overlay a view over the `MessagesCollectionView`. See #755

Reply:
@SD10 🤔 Emm, how can we use `additionalTopInset`? If user set `additionalTopInset`, how we handle this? add to `contentInset`? maybe I think we don't need this currently.